x-pack/legacy/plugins/lens/public/indexpattern_plugin/operations/definitions/cardinality.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { DataType } from '../../../types';
+import { OperationDefinition } from '.';
+import { FieldBasedIndexPatternColumn } from './column_types';
+
+const supportedTypes = new Set(['string', 'boolean', 'number', 'ip', 'date']);
+
+const SCALE = 'ratio';
```

Comment:
I'm not sure `ratio` is the scale I expected, it could also be `ordinal`?

Reply:
I thought so, too, but ratio works and seems to give the correct values, while ordinal gives us an error that I didn't take the time to diagnose. Are there any drawbacks to using ratio here?

Reply:
`ratio` is correct because the cardinality of the field can be interpreted as a ratio (as in this field has double the cardinality compared to this other field). `ratio` also includes all `interval` and `ordinal` usages, those scales can be thought of nested sets - `ratio` is the inner most, `ordinal` is the outer most. This is a nice explanation: https://www.mymarketresearchmethods.com/types-of-data-nominal-ordinal-interval-ratio/